Gemfile

```diff
@@ -0,0 +1,83 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.5.1'
+
+gem 'aws-sdk-s3', '~> 1.46'
+gem 'bootstrap', '~> 4.3', '>= 4.3.1'
+gem 'devise', '~> 4.7'
+gem 'faker', '~> 1.6', '>= 1.6.3'
+gem 'figaro', '~> 1.1', '>= 1.1.1'
+gem 'image_processing', '~> 1.9', '>= 1.9.3'
+gem 'jquery-rails', '~> 4.3', '>= 4.3.5'
+gem 'omniauth', '~> 1.9'
```

Comment:
Here instead of using `omniauth`, you can use `omniauth-facebook` gem. The `omniauth-facebook` gem was designed to use the Facebook OAuth2 strategy.